### DISCUSSION TITLE
REST API v3 Projects API via Personal Access Token

### DISCUSSION THREAD

**Discussion by yairgo** [community/community#24456] [Apps, API and Webhooks]
<p>Is it possible to access these resouces via a personal access token?</p>
<p>- <a href="https://developer.github.com/v3/projects/cards/" rel="nofollow noopener">https://developer.github.com/v3/projects/cards/</a></p>
<ul>
<li><a href="https://developer.github.com/v3/projects/columns/" rel="nofollow noopener">https://developer.github.com/v3/projects/columns/</a></li>
</ul>
<p>If so, what permissions do I need to have on my PAT, and whate permissions do I need to have at the Org/Project/Repo level?</p>
<p>I’m currently getting this: </p>
<pre><code>https://(removed)/api/v3/projects/41/columns: 403 Must have admin rights to Repository. []
</code></pre>
<p>I currently am an Admin at the organization level.</p>
<p>The project I am trying to access has <code>Organization member permission</code> set to admin.</p>
<p>There are 5 repos linked to the project, each project has a team that is a collaborator that I am a part of, and the team has admin rights.</p>
<p>Any help trying to figure this out is greatly appreciated.</p>
<p>-----EDIT-----</p>
<p>I’ve narrowed this down to </p>
<pre><code>https://(removed)/api/v3/orgs/%v/projects
</code></pre>
<p>Returning me back an empty array. </p>
<p>I found a CardID and tried this:</p>
<pre><code>https://(removed)/api/v3/projects/columns/cards/%v
</code></pre>
<p>and recieved <code>403 Must have admin rights to Repository.</code></p>
<p>I’m starting to think this functionality does not work the way I think it should.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/yairgo">@yairgo</a>,</p>
<p>I’ve read through the information that you’ve provided. Could you give a bit more background context for what you’re trying to accomplish? Maybe I can put together a test case and help you out.</p>
<p>Thanks! </p>

**Comment by yairgo** [comment, marked as answer]
<p>I ended up figuring this out. There was a combination of not having correct access rights and me trying to construct urls by hand. The ID for <code>ogs/(org name)/projects/:project_id</code> is not the same as the project_id used <code>/api/v3/projects/:project_id/columns</code>.<br>
So I kept getting unauthorized errors when I was not expecting them.</p>
